### ISSUE TITLE
How to get sota results？

### ISSUE THREAD

**Issue by zzzqqqyyy** [A1ecm/dragonnet-reproduced#1]
Does this code reproduce the sota results? How to get sota results, if possible. Thanks!

**Comment by alecmn** [comment]
This code reproduces the [original dragonnet implementation](https://github.com/claudiashi57/dragonnet). We documented our process [here](https://noiseless-thursday-a1a.notion.site/Reproduction-Analysis-Adapting-Neural-Networks-for-the-Estimation-of-Treatment-Effects-f41ce9a33478407ba8fb601ed4bfa08c). Lmk if you have any more questions.

**Comment by zzzqqqyyy** [comment]
Thanks for your reply, also I have a question.
I found that the propensity score does not weight the prediction results except in the loss function with binary cross entropy optimization, so what is the role of this propensity score estimator? It is also the t_predictions in the code, which the paper authors don't seem to have a good explanation for, and I don't find where the weighting of the inverse propensity weighting method is reflected? 
If you are clear, I hope to get your reply, thanks!

**Comment by helenehanyu** [comment]
I run the exact code in this repo. Got the following result 

```
The back door adjustment result is below
{'tarnet': {'baseline': 0.12606428405314873, 'targeted_regularization': 0.14893175287407603}, 'dragonnet': {'baseline': 0.15822664245695203, 'targeted_regularization': 0.7349647229653203}, 'nednet': {'baseline': 0, 'targeted_regularization': 0}}
the tmle estimator result is this
{'tarnet': {'baseline': 0.1638519517787089, 'targeted_regularization': 0.12792094758497893}, 'dragonnet': {'baseline': 0.22504514604402334, 'targeted_regularization': 0.1282544095818298}, 'nednet': {'baseline': 0, 'targeted_regularization': 0}}
```

I am wondering if anyone knows what might gone wrong..
